src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -983,16 +991,30 @@ public void updateControlledGcsBucket(
    * @param updateParams resource properties to update
    */
   public void updateReferencedBigQueryDataTable(
-      UUID workspaceId, UUID resourceId, UpdateResourceParams updateParams) {
+      UUID workspaceId, UUID resourceId, UpdateReferencedBqTableParams updateParams) {
     // convert the CLI object to a WSM request object
-    UpdateDataReferenceRequestBody updateRequest =
-        new UpdateDataReferenceRequestBody()
-            .name(updateParams.name)
-            .description(updateParams.description);
+    UpdateBigQueryDataTableReferenceRequestBody updateRequest =
+        new UpdateBigQueryDataTableReferenceRequestBody()
+            .name(updateParams.getResourceParams().name)
+            .description(updateParams.getResourceParams().description);
+    if (updateParams.updateReferenceTarget()) {
```

Comment:
nit: I think this method name is slightly misleading, since it's really just checking to see if any resource-specific properties were set. Maybe `hasNewReferenceTargetFields` or something else?